### ISSUE TITLE
Vowels and consonants in viIPA.

### ISSUE THREAD

**Issue by drlor2k** [v-nhandt21/ViMFA#2]
hello @v-nhandt21 , I have a question, please help me!

Based on the following quote: 

**A universal monosyllabic phoneme system has "C(m)-V-C(n)" (m,n >= 0) phoneme patterns.**

Based on the [MFA author's dictionary](https://github.com/MontrealCorpusTools/mfa-models/blob/main/dictionary/vietnamese/ho_chi_minh_city_mfa/vietnamese_ho_chi_minh_city_mfa.dict) I was able to find a list of vowels and consonants. 

With `viIPA`, do you have a specific list for vowels and consonants?

**Comment by v-nhandt21** [comment]
> hello @v-nhandt21 , I have a question, please help me!
> 
> Based on the following quote:
> 
> **A universal monosyllabic phoneme system has "C(m)-V-C(n)" (m,n >= 0) phoneme patterns.**
> 
> Based on the [MFA author's dictionary](https://github.com/MontrealCorpusTools/mfa-models/blob/main/dictionary/vietnamese/ho_chi_minh_city_mfa/vietnamese_ho_chi_minh_city_mfa.dict) I was able to find a list of vowels and consonants.
> 
> With `viIPA`, do you have a specific list for vowels and consonants?

Yep, for Viphoneme, you can get list of phoneme by: https://pypi.org/project/viphoneme/1.0.5/ 

![image](https://github.com/user-attachments/assets/cbc59c26-4a32-4813-821a-d16c37952b6a)

**Comment by drlor2k** [comment]
hi @v-nhandt21, I have some questions:

1. Each phonetic word in your IPA system has only one vowel, right?
2. If I merge the tone with the vowel to create another `viIPA.txt` (ie bận -> b ɤ̆ n 6 -> b ɤ̆6 n), will the whole process of viMFA be affected?
3. Because [the author's MFA for Vietnamese](https://github.com/MontrealCorpusTools/mfa-models/tree/main/acoustic/vietnamese/mfa/v3.0.0) is quite limited in terms of vocabulary, I want to train on my data set. I see version 3.0 of the original author training on a dataset of about 40 hours. However, my dataset is only about 6 hours. Does large and small data affect the final `TextGrid` quality?

Thank you :3

**Comment by v-nhandt21** [comment]
> hi @v-nhandt21, I have some questions:
> 
> 1. Each phonetic word in your IPA system has only one vowel, right?
> 2. If I merge the tone with the vowel to create another `viIPA.txt` (ie bận -> b ɤ̆ n 6 -> b ɤ̆6 n), will the whole process of viMFA be affected?
> 3. Because [the author's MFA for Vietnamese](https://github.com/MontrealCorpusTools/mfa-models/tree/main/acoustic/vietnamese/mfa/v3.0.0) is quite limited in terms of vocabulary, I want to train on my data set. I see version 3.0 of the original author training on a dataset of about 40 hours. However, my dataset is only about 6 hours. Does large and small data affect the final `TextGrid` quality?
> 
> Thank you :3

Hi @drlor2k ,
1. Yep :)) 
2. I think  "b ɤ̆6 n" seems to be more effective than "b ɤ̆ n 6" 
3. I am not sure about the size and quality of your dataset, hihi, so try to use as much as possible.
